dotCMS/src/main/java/com/dotcms/integritycheckers/IntegrityUtil.java

```diff
@@ -625,4 +718,48 @@ public boolean completeCheckIntegrity(final String endpointId) throws Exception
 
         return existConflicts;
     }
+
+
+    public static class IntegrityDataControl implements Serializable {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)